common/views/components/FooterNav/FooterNav.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import {spacing, font} from '../../../utils/classnames';
+
+type navLink = {|
+  href: string,
+  title: string
+|}
```

Comment:
Don't we have a type like this somewhere?

Reply:
Should be capitalised too.

Reply:
Oh yeah, we do - will use that